go.mod

```diff
@@ -1,6 +1,8 @@
 module github.com/walterwanderley/sqlc-connect
 
-go 1.21
+go 1.23
+
+replace github.com/walterwanderley/sqlc-grpc => ../sqlc-grpc
```

Comment:
Remove this line and upgrade to the latest version

Reply:
Done